obspy/core/util/__init__.py

```diff
@@ -34,7 +34,8 @@
                                   PROJ4_VERSION)
 from obspy.core.util.misc import (BAND_CODE, CatchOutput, complexify_string,
                                   guess_delta, loadtxt, score_at_percentile,
-                                  to_int_or_zero, SuppressOutput)
+                                  to_int_or_zero, SuppressOutput,
+                                  _seed_id_map)
```

Comment:
Any reason to put this here? It could just be directly imported from misc.

Reply:
Hmh, it has a leading underscore, so its private, but I thought it might even be of use outside obspy. And even inside obspy its easier to import from core.util instead of core.util.misc. What do you think?

Reply:
Our current policy is that everything with a leading underscore is internal only and we are free to change the API of those functions and even remove them so I'd advise against using it outside of ObsPy.

And these "import aliases" just make it harder to find where things originate from and the convenience improvement from not having to write `.misc` is IMHO not worth the trade-off. But that is just my opinion and feel free to choose.

Reply:
> Our current policy is that everything with a leading underscore is internal only and we are free to change the API of those functions and even remove them so I'd advise against using it outside of ObsPy.

On the other hand, if you do want to use it, you can rename it and remove the leading underscore. But then we need proper tests for it as well (I didnt check if there already are tests for this, maybe there are already).